src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseAuditing.md

```diff
@@ -0,0 +1,124 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+ms.assetid: 14814BF3-51AF-4E51-A8A6-661825BD88D1
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmSqlDatabaseAuditing
+
+## SYNOPSIS
+Gets the auditing settings of an Azure SQL database.
+
+## SYNTAX
+
+```
+Get-AzureRmSqlDatabaseAuditing [-ServerName] <String> [-DatabaseName] <String>
+ [-ResourceGroupName] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmSqlDatabaseAuditing** cmdlet gets the auditing settings of an Azure SQL database.
+To use the cmdlet, use the *ResourceGroupName*, *ServerName*, and *DatabaseName* parameters to identify the database.
+
+## EXAMPLES
+
+### Example 1: Get the auditing settings of an Azure SQL database
+```
+PS C:\>Get-AzureRmSqlDatabaseAuditing -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -DatabaseName "Database01"
+DatabaseName           : database01
+AuditAction            : {}
+AuditActionGroup       : {SUCCESSFUL_DATABASE_AUTHENTICATION_GROUP, FAILED_DATABASE_AUTHENTICATION_GROUP,
+                          BATCH_COMPLETED_GROUP, ...}
+ResourceGroupName      : resourcegroup01
+ServerName             : server01
+AuditState             : Enabled
+StorageAccountName     : mystorage
+StorageKeyType         : Primary
+RetentionInDays        : 0
+```
+
+## PARAMETERS
+
+### -DatabaseName
+SQL Database name.```yaml
```

Comment:
@ranisha2 for the below markdown files, please make sure that the `` ```yaml `` text is on a line by itself (check the `Confirm` and `WhatIf` parameters as examples). This will ensure that the markdown is correctly formatted.